tests/NATS.Client.TestUtilities/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
do we really need this file?

Reply:
no, this was somehow always auto generated, I had removed it and it suddenly appeared again, don't know how this is happening, but I can exclude it.

Reply:
is it Visual studio generating it?

Reply:
could be, I don't know, I will just exclude it from the commit